packages/react-server/core/ClientController.js

```diff
@@ -601,26 +601,35 @@ class ClientController extends EventEmitter {
 			}
```

Comment:
May be easier to look at with [whitespace changes hidden](https://github.com/redfin/react-server/pull/962/files?w=1).